frontend/src/reducers/utils.js

```diff
@@ -0,0 +1,50 @@
+import format from "date-fns/format"
+
+export function calculateNodeStats(state, payload, AmountDataPoints) {
```

Comment:
in JS i would call this `amountDataPoints` -  or maybe `maxLength`?

Reply:
I'll go with maxLength.